stream_alert/alert_processor/main.py

```diff
@@ -114,7 +115,7 @@ def run(loaded_sns_message, context):
         output_dispatcher.dispatch(
             descriptor=descriptor,
             rule_name=rule_name,
-            alert=alert
+            alert=alert['record']
```

Comment:
Can you double check if `rule_name` included in the output? From code, looks it is not.
E.g.
```
"rule_name": "monorail_web_untrusted_commands"
```